src/app/patient/patient.cpp

```diff
@@ -83,7 +83,7 @@ std::vector<std::shared_ptr<Patient>> Patient::getPatient(const std::string& nam
 
 std::shared_ptr<Patient>& Patient::getPatient(const std::string& pesel)
 {
-    auto& patients = Clinic::getPatients();
+    auto patients = Clinic::getPatients();
```

Comment:
Is there any real reason to remove & here?
we do not modify the original Clinic::patients_ vector, so & in my opinion would be safe enough, especially if we use const auto&.

Creating copy here might be quite expensive if we have many patients.
So my advice is to use const auto&